src/amo/components/AddonRecommendations/index.js

```diff
@@ -85,7 +85,7 @@ export class AddonRecommendationsBase extends React.Component<Props> {
     if (newRecommendations && oldRecommendations !== newRecommendations) {
       const { fallbackReason, loading, outcome } = newRecommendations;
 
-      if (loading) {
+      if (loading || this.props.errorHandler.hasError()) {
```

Comment:
You could destructure `errorHandler` from `this.props` at the top of this function where other variables are being destructured (e.g., at line 72).